### PR TITLE
fix: add navigation bar for the kurtosis frontend

### PR DESCRIPTION
**Pull request by Peeeekay**
Added navigation bar for users to easily navigate between pages; also add title bar and improved usability as well.

### CODE REVIEW

File: engine/server/webapp/asset-manifest.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "files": {
-    "main.css": "/static/css/main.7b6afa70.css",
-    "main.js": "/static/js/main.2f3b687c.js",
+    "main.css": "/static/css/main.1fada45f.css",
```
Comment (h4ck3rk3y) [inline engine/server/webapp/asset-manifest.json:3]: can we nuke the previous css files? i see new files being added but the old ones aren't being removed!

also curious whether these assets should be in `.gitignore` if they are generated assets

File: engine/frontend/src/component/ServiceInfo.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                </div>  
+            </div>                    
+            <RightPanel home={false} isServiceInfo={true} enclaveName={enclaveName}/>
+        </div>
+    )
+}
+
+export default ServiceInfo;
+
+{/* <div className="flex w-[calc(100vw-24rem)] flex-col bg-white">
```
Comment (h4ck3rk3y) [inline engine/frontend/src/component/ServiceInfo.js:114]: perhaps nuke

File: engine/frontend/src/component/CreateEnclave.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        <div className='h-full w-full flex'>
+            <Routes>
+                <Route path="/create" element={<CreateEnclaveModal name={name} setName={setName} args={args} setArgs={setArgs} handleSubmit={handleModalSubmit}/>}/>
+                <Route path="/progress" element={<CreateEnclaveView args={args} packageId={name} enclaveInfo={enclaveInfo}/>} />
+            </Routes>
+        </div>
+    )
+}
+
+export default CreateEnclave;
```
Comment (h4ck3rk3y) [inline engine/frontend/src/component/CreateEnclave.js:28]: perhaps new line here and elsewhere?

File: engine/frontend/src/component/LogView.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                        {log}
+                        </li>
+                    ))}
+                </ul>
+            </div>
+        </div>    
+    )
+}   
+
+{/* </div>
```
Comment (h4ck3rk3y) [inline engine/frontend/src/component/LogView.js:20]: perhaps delete this commented code

**Review by h4ck3rk3y — APPROVED** [review]
How thorough should I be here? I haven't very deeply looked into stuff under src/ as far as I understand this is still in beta and you are iterating on it quite quickly.

File: engine/frontend/src/App.js [hunk 5]
```diff
@@ -1,23 +1,14 @@
 import './App.css';
-import Home from './components/Home';
-import Services from "./components/Services"
-import ServiceInfo from "./components/ServiceInfo"
-import CreateEnclave from './components/CreateEnclave';
-import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Home from './component/Home';
```
Comment (h4ck3rk3y) [inline engine/frontend/src/App.js:2]: Are there any tests for any of this? If not that's alright. Can you link some screenshots from this build in the comments?

File: engine/server/webapp/asset-manifest.json (hunk 1 above)
Comment (Peeeekay) [reply engine/server/webapp/asset-manifest.json:3]: not yet; i have not added frontend to build pipeline so need these static files; will nuke it after doing so.. in the pipeline :D

File: engine/frontend/src/App.js (hunk 5 above)
Comment (Peeeekay) [reply engine/frontend/src/App.js:2]: yepp; can do that. no tests atm.. will keep the commented code ( as will uncommenting it upcoming PRs)

**Comment by Peeeekay** [comment]
added loom video @h4ck3rk3y https://www.loom.com/share/e22041c42c514af8bfb2d3ce68cc2993?sid=184037a9-4119-4b49-be31-d6b3c4d5f5e2

**Pull request merged**